src/context/Workflow.tsx

```diff
@@ -19,4 +19,17 @@ const useEntityWorkflow = () => {
     return useContext(WorkflowContext);
 };
 
-export { WorkflowContextProvider, useEntityWorkflow };
+const useEntityWorkflow_Editing = () => {
```

Comment:
Little helper hook that I think can be used all over the app to fetch if workflows are in edit. I did not switch all components to this to prevent needing to test too much

Reply:
Name alternatives to consider: `useEntityWorkflow_Edit`, `useWorkflow_EntityEdit`, or `useEntityEditWorkflow`. _Thinking out loud._